tools/make/diplomat-coverage/src/allowlist.rs

```diff
@@ -391,6 +389,17 @@ lazy_static::lazy_static! {
         "icu::calendar::cal::GregorianDateInner",
         "icu::calendar::any_calendar::AnyDateInner",
 
+        // Generic and UTF-8 segmenter types
+        "icu::segmenter::grapheme::GraphemeClusterBreakIterator",
```

Comment:
will clash: https://github.com/unicode-org/icu4x/pull/6441

Reply:
Ah, probably should have asked you to just leave those types alone, that rebase was rather annoying.